src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net7.Manifest/Microsoft.NET.Workload.Mono.Toolchain.net7.Manifest.pkgproj

```diff
@@ -38,11 +38,6 @@
       <_WorkloadManifestValues Include="WorkloadVersion" Value="$(PackageVersion)" />
       <_WorkloadManifestValues Include="PackageVersionNet7" Value="$(PackageVersionNet7)" Condition="'$(PackageVersionNet7)' != ''" />
       <_WorkloadManifestValues Include="PackageVersionNet7" Value="$(PackageVersion)" Condition="'$(PackageVersionNet7)' == ''" />
-
```

Comment:
Should we have `Microsoft.NET.Workload.Mono.Toolchain.net8.Manifest` also ?

Reply:
Not yet. The `Microsoft.NET.Workload.Mono.Toolchain.Manifest` serves as the one for net8 right now.